packages/v4/src/content/design-guidelines/usage-and-behavior/dashboard/dashboard.md

```diff
@@ -0,0 +1,331 @@
+
+**Capacity**
+- Physical capacity (percentage used)
+- Requested capacity (percentage used)
+
+
+### Step 2: Prioritizing your cards
+
+Prioritize your list by determining which cards contain information most useful to the broadest range of use cases.
+
+Don’t spend too much time on this step; it doesn’t have to be perfect. You might have a clear hierarchy of importance amongst your list, or you may have a list of cards which are of equal importance. If you find yourself struggling, just move on to the next step.
+
+Ideally, all of the information you are displaying is important. The purpose of this step is to help us determine which cards we want front and center, and which we can receive less emphasis . This distinction is primarily important with responsive design. When your four-column layout becomes a one-column layout on a mobile viewport, ordering the cards appropriately will make sure that the most important information comes to the top.
+
+#### Example of a prioritized card list
+
+**Unprioritized:**
+- Top consumers
+- Events
+- Capacity
+- Inventory
+- Performance
+- Details
+- Health
+
+**Prioritized:**
+- Details
+- Performance
+- Inventory
+- Health
+- Capacity
+- Top Consumers
+- Events
+
+Note: The cards at the bottom of our list are not necessarily “unimportant,” they just simply wouldn’t make sense being above the cards at the top of the list.
+
+### Step 3: Sizing your cards
+Next, you must size your cards by asking yourself one question: How big do I think this card will need to be?
+
+Here are a few questions to ask yourself, along with some real-life examples to give you a better sense of how to approach this question.
+
+1. Start by using the three standard sizes: small, medium, large. This sizing will help prepare us for our next step, determining column structure. These sizes refer to the overall surface area of your cards, not specifically the length or the width. You can have a tall medium card and a wide medium card that both take up the same surface area. We are guesstimating. Don’t spend considerable amounts of time determining whether a card is medium or large. This step simply builds a foundation for iteration. We want to start thinking about the layout of each card on a conceptual level, which will help us when we begin to iterate on our layout.
+
+2. Ask questions about each card:
+- What format would this information most clearly be displayed as?
+- How much data will this card contain?
+- Is the amount of information in this card static or dynamic? The size and shape of static cards will not change over time, for example it will always be a card with one chart. A dynamic card could contain a list of items that will grow or shrink, like an events list.
+
+Here are some examples of how to apply these questions to different types of cards to determine their appropriate sizing in your dashboard:
+
+**Details card**
+“What format would most clearly display this information?”
```

Comment:
Need an extra page break here so that the question doesn't appear on the same line as the header